src/js/tracker.js

```diff
@@ -916,6 +923,23 @@
 		}
 
 		/**
+		 * Check that *both* optimizely and optimizely.data exist
+		 *
+		 * @param property optimizely data property
+		 * @param snd optional nested property
+		 */
+		function getOptimizelyXData(property, snd) {
+			var data;
+			if (windowAlias.optimizely && windowAlias.optimizely.data) {
+				data = windowAlias.optimizely.get(property);
+				if (typeof snd !== 'undefined' && data !== undefined) {
+					data = data[snd]
+				}
+			}
+			return data
+		}
```

Comment:
Yes, we're fine removing it. 

Just little a heads-up: are we sure that `window.optimizely.get(nonExistingProperty)` returns `undefined` and not `null`? I'm asking because in JS these are not equivalent: `null === undefined` and `null === null`, so `data !== undefined` may be non-protective enough if `optimizely.get` may return `null` (you may start to wonder how silly JS can be).